.buildkite/test-pipeline.yaml

```diff
@@ -171,7 +171,7 @@ steps:
     - vllm/
     - tests/v1
   commands:
-    - pytest -v -s v1
+    - VLLM_USE_V1=1 pytest -v -s v1
```

Comment:
@WoosukKwon @robertgshaw2-neuralmagic please confirm this is expected

Reply:
oh, the original test wasn't broken. The current test uses `patch()` in the test body to patch the env var. I am proposing to run v1 tests directly with env var set here, so in the future, when we switch to v1, we don't need to alter the test code. 

I can leave this out of the PR too.

Reply:
Make sense to me